common/persistence/operationModeValidator_test.go

```diff
@@ -65,7 +65,7 @@ func (s *validateOperationWorkflowModeStateSuite) TestCreateMode_UpdateCurrent()
 	stateToError := map[enumsspb.WorkflowExecutionState]bool{
 		enumsspb.WORKFLOW_EXECUTION_STATE_CREATED:   false,
 		enumsspb.WORKFLOW_EXECUTION_STATE_RUNNING:   false,
-		enumsspb.WORKFLOW_EXECUTION_STATE_COMPLETED: true,
+		enumsspb.WORKFLOW_EXECUTION_STATE_COMPLETED: false,
```

Comment:
what is the use case for create workflow with completed state?

Reply:
we can potentially optimize the workflow replication / resend case by
* replicate all history events
* write the mutable state